cli/flags.rs

```diff
@@ -448,6 +448,8 @@ pub fn flags_from_vec(
         .collect();
       argv.extend(files);
 
+      argv.push("--write".to_string());
```

Comment:
A word of explanation why we're adding this would be helpful

Reply:
Thanks! Will update.